src/index.html

```diff
@@ -11,7 +11,162 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
     <script type="text/javascript" src="./main.js"></script>
+    <form
+      action="https://mate-academy-form-lesson.herokuapp.com/create-application"
+      onsubmit="onSubmit()"
+      method="post"
+    >
+      <fieldset>
+        <legend>Personal information</legend>
+        <div class="form-field">
+          <label for="surname">Surname:</label>
+          <input type="text"
+            name="surname"
+            id="surname"
+          >
+        </div>
+
+        <div class="form-field">
+          <label for="name">Name:</label>
+          <input type="text"
+            name="name"
+            id="name"
+          >
+        </div>
+
+        <div class="form-field">
+          <label for="age">How old are You?</label>
+          <input type="number"
+            name="age"
+            id="age"
+            value="12"
+            min="1"
+            max="100"
+          >
+        </div>
+
+        <div class="form-field">
+          <label for="birth-date">Full date of birth:</label>
+          <input type="date"
+            name="birth-date"
+            id="birth-date"
+          >
+        </div>
+
+        <div class="form-field">
+          <label for="terms-and-agreements">I accept the term of the agreement</label>
+          <input type="checkbox"
+            name="terms-and-agreements"
+            id="terms-and-agreements"
+            required
+          >
+        </div>
+      </fieldset>
+
+      <fieldset>
+        <legend>Registration</legend>
+        <div class="form-field">
+          <label for="email">E-mail:</label>
+          <input type="email"
+            name="email"
+            id="email"
+            placeholder="email@example.com"
+            required
+          >
+        </div>
+
+        <div class="form-field">
+          <label for="password">Password:</label>
+          <input type="password"
+            name="password"
+            id="password"
+            minlength="5"
+            maxlength="16"
+            required
+          >
+        </div>
+      </fieldset>
+
+      <fieldset>
+        <legend>An interesting fact about you!</legend>
+        <div class="form-field">
+          Do you love cats?
+          <input type="radio"
+            name="love-cats"
+            id="radio1"
+            value="yes"
+          >
+          <label for="radio1">Yes</label>
+
+          <input type="radio"
+            name="love-cats"
+            id="radio2"
+            value="no"
+          >
+          <label for="radio2">No</label>
+        </div>
+
+        <div class="form-field">
+          <label for="favorite-color">What is your favorite color?</label>
+          <input type="color"
+            name="favorite-color"
+            id="favorite-color"
```

Comment:
Use camelCase for id attribute
```suggestion
            id="favoriteColor"
```